_config.yml

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This file is a Jekyll config - it allows for defining variables (strings) that are used throughout the site.